README.md

```diff
@@ -4,7 +4,21 @@ This repo contains example code to help you learn how to use [Kedro](https://git
 
 ## How do I use it?
 
-Each example has an README.md that explains what it does and how to work with it. We recommend you start with `kedro-tutorial`, which is the complete version of the [Space Flights tutorial](https://kedro.readthedocs.io/en/latest/03_tutorial/02_tutorial_template.html) described in the Kedro [documentation](https://kedro.readthedocs.io) and includes the data necessary to run the project.
+Each example has an README.md that explains what each project does and how to work with it. 
+
+## What will I find in this repository?
+
+- We recommend you start with [`kedro-tutorial`](https://github.com/quantumblacklabs/kedro-examples/tree/master/kedro-tutorial), which is the complete version of the [Space Flights tutorial](https://kedro.readthedocs.io/en/latest/03_tutorial/02_tutorial_template.html) described in the Kedro [documentation](https://kedro.readthedocs.io) and includes the data necessary to run the project
+- Documentation that you can use to deliver Kedro training in [`kedro-training`](https://github.com/quantumblacklabs/kedro-examples/tree/master/kedro-training)
+
+## Where can I find more `kedro-examples`?
```

Comment:
This is a good idea, I would also like to create a cool Kedro example in my Github as well :) Just to double-check, there is no problem (legally) to refer to a repo by an external person without letting them know (or they already know?) ?

Reply:
You need to check the license they've used for their repo, to be on the conservative side.